infrastructure/utils.py

```diff
@@ -207,6 +216,7 @@ def find_phase(fields):
             or field.startswith("Adjusted")
             or field.startswith("Original")
             or field.startswith("Budgeted")
+            or field.startswith("Budget year")
```

Comment:
I think this was the bug that stopped uploads from showing up on the muni profile and search results - I think they have filters for this phase